### ISSUE TITLE
Should we have term limits for the AB (and possibly TAG)?

### ISSUE THREAD

**Issue by tantek** [w3c/w3process#165]
As @michaelchampion offhandedly referenced in https://github.com/w3c/w3process/issues/31#issuecomment-301846868, and given that we are moving more actual AB work to public CGs (like the Process CG) where people can keep doing work as they please, it is perhaps time to consider introducing term limits for AB participation, and possibly the TAG. We can do this incrementally with just for the AB at first (e.g. for Process 2019), and assuming that works well, introduce it similarly for the TAG (which may have additional complexities like should term limits apply to appointments also? (likely should))

**Comment by dwsinger** [comment]
random thoughts: I have never really understood term limits for posts like this; is there such a strong incumbent-advantage that better-qualified people are standing but failing to get elected (the problem that term limits purport to solve in public elections)? surveys I have seen also suggest that people want term limits and want to re-elect their own representative, which suggests that they want to use term limits to get rid of other peoples' reps, ahem.

I note however, that the TAG appointees tend to be the same year after year, which suggests that the appointment process isn't being used to get balance, but to retain good people. I have previously wondered why we don't require that appointees in year N stand for election in year N+1...the Director could still appoint them if they fail to get elected, but it gives valuable feedback on how the community feels about someone, and if there is an 'incumbent advantage', then the Director can find someone brilliant but not widely known, appoint in year N, and then they become known as brilliant and stand a much better chance of getting elected in year N+1.

**Comment by michaelchampion** [comment]
There's a dilemma -- the most QUALIFIED people on both the AB and TAG are almost certainly those who've learned how the system works and what the fixable and unfixable problems are.    But both groups need to be stimulated/challenged to look at old problems from fresh perspectives and new problems without old baggage.  

A related problem is that there are highly qualified people willing to contribute to the work of the TAG and AB but don't get elected/re-elected due to the various quirks of any election system.  A process that creates more vacant seats to encourage people to run, and increases the pool of people with AB and TAG experience that they can apply in the community outside the formal advisory groups.

Not to beat my favorite dead horse or anything ;-) but term limits could help mitigate the downside of recent election changes that make it easier for single-issue candidates or those advocating extreme positions appreciated by a fairly small minority of the membership to win AB and TAG elections.   We might well benefit from the diversity of such perspectives in the short run, but ensuring that other perspectives are also cycled in and out would prevent paralysis if partisans of competing points of view are elected.

**Comment by tantek** [comment]
@dwsinger appreciate the random thoughts, here's attempts for a few:

> is there such a strong incumbent-advantage that better-qualified people are standing but failing to get elected (the problem that term limits purport to solve in public elections)

Close but not quite the actual problem we are (anecdotally) hearing.

Actual problem: there such a strong incumbent-advantage that **equivalently-qualified** people are **not bothering to stand** because they 1. (fairly accurately) perceive that incumbent-advantage, and 2. do not wish to suffer the humiliation (peers, professionally, by their employer) of standing and "losing". Whether or not they should feel that (they shouldn't obv) is not the point. The point is it is a human reality, and thus should be considered in our "software for humans" we call the process. Talk to Chris Wilson @cwilso or Andrew Betts if you want a more personal understanding.

> surveys I have seen also suggest that people want term limits and want to re-elect their own representative

You're missing the (likely in several existing examples) people (orgs) that have others they would likely run if/when a "representative" termed out. 

> why we don't require that appointees in year N stand for election in year N+1...the Director could still appoint them if they fail to get elected

yes to your line of thinking here. makes complete sense, please file a separate issue to track that (perhaps something like "Should TAG appointees be automatically nominated for the next TAG election?")

**Comment by dwsinger** [comment]
see #166 ...

**Comment by tantek** [comment]
@michaelchampion makes additional good points, that I'd like to expand on:

> There's a dilemma -- the most QUALIFIED people on both the AB and TAG

Yes to this entire paragraph, AND a system with term limits would strongly incentivize those with more experience (and desire to keep applying it) to mentor and knowledge transfer to newer members, and that can only be a good thing (broadens / strengthens institutional memory).

> Not to beat my favorite dead horse or anything ;-)

Pretty much agreed with all that too. And it's not dead, not as long as we have contradiction in the process, and the possibility of electing "No other candidate" (hence the reason for keeping those two in the process).

**Comment by dwsinger** [comment]
This seems like a policy issue to start with. Referring to the AB (and they should probably call in the AC).

**Comment by tantek** [comment]
I'm happy to continue discussion in the AB, but I'd prefer to keep this issue open to gather any public comments / concerns.

**Comment by tantek** [comment]
I’m happy to continue the discussion of @W3C AB term limits in the AB itself, however I’d prefer to keep this issue open to gather public feedback, comments, support, concerns, etc.

(Originally published at: http://tantek.com/2018/052/t1/ab-term-limits-issue)

**Comment by dwsinger** [comment]
yes, leaving open here as a possibility for future consideration.

**Comment by dwsinger** [comment]
we might also consider whether term limits for elected and appointed should be the same, or could be different.

**Comment by dwsinger** [comment]
AB's current thinking is maybe 3-4 terms, but we want to understand what we're trying to optimize for (the 'why?')

**Comment by LJWatson** [comment]
It seems to me, that we want an AB (or a TAG) with a balance of people with historical knowledge and experience, and people with new perspectives and experiences. If we exclude either, we're likely to be missing something valuable.

Anecdotal feedback is useful, but it also helps to have some actual data, so I did some archeology and pulled together the following information about the AB over the past 20 years.

41 people have served on the AB in the last 20 years (counting from the 1998 election to the present):
* 19 people (46%) served 1 term or less.
* 8 people (19.5%) served 2 terms
*  4 people (10%) served 3 terms or less.
* 4 people (10%) served 4 terms.
* 3 people (7%) served 5 terms.
* 1 person (2.5%) served 6 terms.
* 1 person (2.5%) served 7 terms.
* 1 person (2.5%) served 8 terms.

This means that 75% of people served 3 terms or less, or to put it another way, the majority naturally met the suggested term limit.

The current AB is a remarkably similar reflection of the historical data:
* 2 people have served 1 term or less.
* 3 people have served 2 terms
* 2 people have served 3 terms or less.
* 2 people have served 5 terms

In other words, 78% of the current AB have served for 3 terms or less.

**Comment by michaelchampion** [comment]
“Anecdotal feedback is useful, but it also helps to have some actual data,”

Thanks for the data, They don't address @tantek and @frivoal’s point that eminently qualified people are not bothering to run because incumbents who run usually get elected.   

From a quick scan of AB election results, incumbents who run usually win.  The years such as 2013 and 2014 where incumbents lost elections are also the years where there were the most candidates. 

@frivoal notes that term limits should not imply a lifetime ban on running again. Right, the proposals I’ve seen just ask long term incumbents to sit out an election or a term.

**Comment by dwsinger** [comment]
Closing as we are not reaching consensus, and term limits have significant drawbacks.

**Comment by frivoal** [comment]
@tantek I am compiling a DoC for this year's process iteration. We resolved (at the Tokyo AB meeting) against your suggestion to introduce term limits in Process 2019. I'd appreciate if you could confirm you can live with that resolution.
